housingResidentInformationAPI.Tests/V1/Gateways/housingGatewayTests.cs

```diff
@@ -4,34 +4,34 @@
 using AutoFixture;
 using FluentAssertions;
 using NUnit.Framework;
-using UHResidentInformationAPI.Tests.V1.Helper;
-using UHResidentInformationAPI.V1.Domain;
-using UHResidentInformationAPI.V1.Enums;
-using UHResidentInformationAPI.V1.Factories;
-using UHResidentInformationAPI.V1.Gateways;
-using UHResidentInformationAPI.V1.Infrastructure;
-using Address = UHResidentInformationAPI.V1.Infrastructure.Address;
-using DomainAddress = UHResidentInformationAPI.V1.Domain.Address;
-
-namespace UHResidentInformationAPI.Tests.V1.Gateways
+using housingResidentInformationAPI.Tests.V1.Helper;
+using housingResidentInformationAPI.V1.Domain;
+using housingResidentInformationAPI.V1.Enums;
+using housingResidentInformationAPI.V1.Factories;
+using housingResidentInformationAPI.V1.Gateways;
+using housingResidentInformationAPI.V1.Infrastructure;
+using Address = housingResidentInformationAPI.V1.Infrastructure.Address;
+using DomainAddress = housingResidentInformationAPI.V1.Domain.Address;
+
+namespace housingResidentInformationAPI.Tests.V1.Gateways
 {
     [TestFixture]
-    public class UHGatewayTests : DatabaseTests
+    public class HousingGatewayTests : DatabaseTests
```

Comment:
I think this should still be UHGateway and UHGatewayTests, again because that's what the database is called. Housing is the name of our domain, but the gateway is specifically to access the database